test/test_loader.py

```diff
@@ -62,16 +62,16 @@ def bot_command():
     return bot_command
 
 
+TMP_CONFIG = """
+[core]
+owner=testnick
```

Comment:
Is there a test-related reason to keep this without spaces around `=`, or can we just make it nice-nice?

Reply:
Oh, we can totally make it nice-nice!